src/main/webapp/js/build.js

```diff
@@ -5,7 +5,7 @@ if (rebuildButton) {
     event.preventDefault();
     const buildUrl = `${rebuildButton.dataset.buildPath}?delay=0sec`
     if (rebuildButton.dataset.parameterized === 'true') {
-      window.location.href = buildUrl
+      doParameterizedRebuild();
```

Comment:
I think we can get rid of the javascript and move this all to the server side now that we're using replay action

Reply:
@timja I prefer to use the flexibilty of javascript. In the last implementaiton I use the popup, build scheduled, I will add some error popup in case of mistake. This make things nicer